app/web_framework/cluster_master_application.py

```diff
@@ -60,11 +60,27 @@ def initialize(self, route_node=None, cluster_master=None):
         super().initialize(route_node)
 
 
-class _RootHandler(_ClusterMasterBaseHandler):
+class _ClusterMasterBaseStaticFileHandler(ClusterBaseHandler, tornado.web.StaticFileHandler):
```

Comment:
I like the multiple inheritance approach so that all the handlers can share exception handling. How do you feel about keeping this a part of `_BuildResultHandler` for now?

I suggest that just because that single handler is the only route uses this pattern, so it's a bit hard to guess what abstraction would be useful if we ever had another static file handler. And also I think the inheritance chain of all these handlers has gotten complicated enough that it might be easier to comprehend (for now) if we just kept all this inside `_BuildResultHandler` like 
```python
class _BuildResultHandler(ClusterBaseHandler, tornado.web.StaticFileHandler):
...
```

What do you think?